.github/workflows/mobile-android_build.yml

```diff
@@ -165,12 +192,13 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
perhaps this could be empty - the env var would never be normally - ill test

Reply:
tested in staging repo - this is never empty - no matter who opens a PR (and what creds the token has)